### PR TITLE
Add types declaration file

### LINKED ISSUE #2: TypeScript types declaration file
If you want I can write a TypeScript types declaration file for this package so that TS developers can utilise the type-checking feature of the language as they use this package. Let me know if you are interested.

### PR DESCRIPTION
**Pull request by geopic**
Resolves #2

### CODE REVIEW

**Comment by lahmatiy** [comment]
Could you please address additional conditions (see readme):
- `align` and `truncate` options can take `'left'` or `'right'` values only
- `ellipsis` option can take a falsy value (`false`, `null` ...) to disable ellipsis

**Comment by geopic** [comment]
Sure thing @lahmatiy, I'll start on it and make another commit.

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+declare module 'fixed-width-string' {
+  export default function (str: string, width: number, options?: {
+    padding?: string;
+    align?: 'left' | 'right';
+    ellipsis?: string | false | null;
+    truncate?: string;
```
Comment (lahmatiy) [inline index.d.ts]: Should be:
```suggestion
    truncate?: 'left' | 'right';
```

**Comment by lahmatiy** [comment]
Thank you!

**Comment by lahmatiy** [comment]
Published as [2.0.0](https://github.com/lahmatiy/fixed-width-string/releases/tag/v2.0.0)

**Pull request merged**